packages/python-packages/doc-warden/warden/index_packages.py

```diff
@@ -280,7 +280,9 @@ def render(config, pkg_list):
         'python': OUTPUT_TEMPLATE,
         'js': OUTPUT_TEMPLATE,
         'java': JAVA_OUTPUT_TEMPLATE,
-        'net': OUTPUT_TEMPLATE
+        'net': OUTPUT_TEMPLATE,
```

Comment:
Don't you also need to add handling for them? Or if there isn't any special handling perhaps we should just allow any language to come through instead of maintaining a hard-coded list.

Reply:
> perhaps we should just allow any language to come through instead of maintaining a hard-coded list.

Yep I was thinking the same thing. In fact I'm not quite sure why it is different for Java.

Reply:
Actually I think I'm going to need to have support for swift here. Looks like Doc Warden has logic for processing the source files to extract package names etc, therefore I'll need to crack open the project files in doc warden.

Reply:
Turns out doc-warden's ability to correctly process README files is tightly coupled to its ability to parse and understand package information for any given language. As a result I needed to implement logic for swift specifically.